### PR TITLE
insync: 3.8.6.50504 -> 3.8.7.50507

### PR DESCRIPTION
**Pull request by tuergeist**
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [x] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/applications/networking/insync/default.nix [hunk 1]
```diff
@@ -47,7 +47,8 @@ let
     src = fetchurl {
       # Find a binary from https://www.insynchq.com/downloads/linux#ubuntu.
       url = "https://cdn.insynchq.com/builds/linux/insync_${version}-lunar_amd64.deb";
-      sha256 = "sha256-BxTFtQ1rAsOuhKnH5vsl3zkM7WOd+vjA4LKZGxl4jk0=";
+      sha256 = "sha256-Fu3tmnYhcm6xIEnkTtpmMJmEgIUnZdLsAtmIktWdrRs=";
+      #sha256 = lib.fakeSha256;
```
Comment (betaboon) [inline pkgs/applications/networking/insync/default.nix:51]: this shouldn't be here.

File: pkgs/applications/networking/insync/default.nix [hunk 2]
```diff
@@ -124,3 +125,4 @@ in buildFHSEnvBubblewrap {
   # Since "insync start" command starts a daemon, this daemon should die with it.
   dieWithParent = false;
 }
+
```
Comment (betaboon) [inline pkgs/applications/networking/insync/default.nix:128]: please don't add a tailing newline

**Comment by hellwolf** [comment]
Hi thanks for the PR. There is a bug needs to be fixed first: #279426

**Comment by hellwolf** [comment]
Hi, I believe we are already at 3.8.7.50516

Btw, also I have a new package: https://github.com/NixOS/nixpkgs/pull/302422

Please give it some support by clicking like :+1:  in order for faster merging!

**Pull request closed without merging**